src/packages/@ncigdc/components/Modals/GroupValuesModal.tsx

```diff
@@ -344,53 +347,82 @@ export default compose(
                           }), {}),
                         });
                       }}
-                      style={{ backgroundColor: group.every((binKey: string) => selectedGroupBins[binKey]) ? '#d5f4e6' : '' }}
-                    >
+                      style={{
+                        backgroundColor: group.every((binKey: string) => selectedGroupBins[binKey]) ? '#d5f4e6' : '',
+
+
+                      }}
+                      >
                       {group.length > 1 || group[0] !== groupName
                         ? (
                           <EditableLabel
-                            containerStyle={{ justifyContent: 'flex-start' }}
-                            handleSave={(value: string) => setCurrentBins({
-                              ...currentBins,
-                              ...group.reduce((acc: ISelectedBinsProps, bin: string) => ({
-                                ...acc,
-                                [bin]: {
-                                  ...currentBins[bin],
-                                  groupName: value,
-                                },
-                              }), {}),
-                            })
+                            buttonDisplayed={false}
+                            containerStyle={{
+                              justifyContent: 'flex-start',
+                            }}
+                            handleSave={(value: string) => {
+                              if (some(currentBins, (bin: IBinProps) => bin.groupName.trim() === value.trim())) {
```

Comment:
Author comment: I trim the space during the comparing so that groupName like "name " and "name" are same name, and they can't existed at the same time.